### PR TITLE
Split up our css code into multiple files

### PR DESCRIPTION
**Pull request by Henni**
The code is pretty messy. We should clean it up after the next release:
-> use classes instead of directive names
-> order the rules and structure the rules hierarchically (scss nesting)

### CODE REVIEW

File: css/public/style.scss [hunk 1]
```diff
@@ -1,841 +0,0 @@
...
-	padding: 0;
-	flex-wrap: wrap;
-	justify-content: space-around;
-}
-avatar.maximized .avatar-options > [class^="icon-"] {
-	min-width: 25%;
-	display: block;
-}
-
-#check-all + label:before {
```
Comment (Henni) [inline css/public/style.scss]: This is the only rule I removed. I wasn't able to find any place it was used (it was added by me in one of the initial commits)

File: css/_app-content-list.scss [hunk 2]
```diff
@@ -0,0 +1,127 @@
+/* app content list & detail view */
+/* TO BE MOVED TO CORE apps.css ONCE STANDARDIZED ACROSS APPS */
+/* DO NOT MAKE ANY CHANGES SPECIFIC TO CONTACTS HERE! */
+
```
Comment (Henni) [inline css/_app-content-list.scss]: @skjnldsv please take a look at this part.
Please verify that all of this code is part of the app-content-list, that should be moved to core and that nothing is missing.

Comment (skjnldsv) [reply css/_app-content-list.scss]: It's planned. I'm working on it!

File: css/style.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@import "app-content-list";
+
```
Comment (skjnldsv) [inline css/style.scss]: typo?

File: css/_settings.scss [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+  transition: 500ms all ease-in-out;
+}
+#importscreen-sidebar-block {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: $color-main-background;
+  z-index: 500;
+  opacity: 0.5;
+}
```
Comment (skjnldsv) [inline css/_settings.scss]: please add correct EOF

File: css/_navigation.scss [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+/* Contacts List */
+#new-contact-button {
+  margin: 14px auto; /* to have the same height than a contact*/
+  width: calc(100% - 20px) !important;
+  text-align: left;
+  padding-left: 34px;
+  background-position: 10px center;
+}
```
Comment (skjnldsv) [inline css/_navigation.scss]: please add correct EOF

File: css/_details.scss [hunk 6]
```diff
@@ -0,0 +1,377 @@
...
+.select2-container.select2-container-multi .select2-choices span {
+  display: flex;
+  flex-wrap: wrap;
+}
+.select2-container-multi .select2-choices .select2-search-choice .select2-search-choice-close {
+  display: block !important;
+  right: 4px;
+  left: auto;
+  top: 7px
+}
```
Comment (skjnldsv) [inline css/_details.scss]: please add correct EOF

File: css/style.scss [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+@import "details";
+@import "contactlist";
+@import "navigation";
+@import "settings";
```
Comment (skjnldsv) [inline css/style.scss]: please add correct EOF

File: css/style.scss (hunk 3 above)
Comment (Henni) [reply css/style.scss]: What do you mean?

Comment (skjnldsv) [reply css/style.scss]: The new line! :)

**Pull request merged**